src/api/movies.tsx

```diff
@@ -0,0 +1,8 @@
+const BASE_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=9f56fe16';
```

Comment:
Change it to https to make your github pages work
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=9f56fe16';
```